### DISCUSSION TITLE
Tracking Discussion for Validation Rules

### DISCUSSION THREAD

**Discussion by JSv4** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#412] [Ideas]
Please add each proposed validation rule as a separate comment

**Comment by arthur-clara** [comment]
If we end up implementing authorized shares at the issuer and class level with the logic proposed in Issue 415 (one of: numeric value, "NOT APPLICABLE", "UNLIMITED"), we need to think whether certain combinations of variables make sense. e.g. "UNLIMITED" issuer + "UNLIMITED" class or "UNLIMITED" issuer + "NON APPLICABLE" class.
